deepchem/splits/splitters.py

```diff
@@ -159,23 +144,23 @@ def train_valid_test_split(self,
       The fraction of data to be used for the test split.
     seed: int, optional (default None)
       Random seed to use.
-    log_every_n: int, optional
+    log_every_n: int, optional (default 1000)
       Controls the logger by dictating how often logger outputs
       will be produced.
 
     Returns
     -------
-    Train and test datasets as dc.data.Dataset objects.
+    Tuple[Dataset, Dataset, Dataset]
```

Comment:
Could you add a usage example for this and the other splitters? This is optional for this PR, but would be very helpful for our users!